src/Database/Query.php

```diff
@@ -658,7 +657,7 @@ public static function parseQueries(array $queries): array
             try {
                 $parsed[] = Query::parse($query);
             } catch (\Throwable $th) {
-                throw new Exception("Invalid query: ${query}", previous: $th);
+                throw new DatabaseException("Invalid query: ${query}");
```

Comment:
We don't need to expose it. An end user cannot control the query ( or fix any syntax issues on their query )